src/DevTools/flushPromiseQueue.ts

```diff
@@ -2,5 +2,5 @@
  * test utility to wait for all promises to be resolved
  */
 export function flushPromiseQueue() {
-  return new Promise(r => setTimeout(r, 0))
+  return new Promise(jest.requireActual("timers").setImmediate)
```

Comment:
Looks like this is causing a spec failure — Needed to be done to support usage in specs with fake timers.